src/templates/pad.html

```diff
@@ -400,6 +401,7 @@ <h2 data-l10n-id="pad.share.emebdcode"></h2>
 
               var plugins = require('ep_etherpad-lite/static/js/pluginfw/client_plugins');
               var hooks = require('ep_etherpad-lite/static/js/pluginfw/hooks');
+              var padutils = require('../static/js/pad_utils');
```

Comment:
Is this require() needed?

Reply:
I guess it's not (only the first one seems to be needed).

Reply:
Hi @muxator ,

Apologies I didn't notice that was still in there, that 2nd one is a result of testing and is not effective. Only the first 'require' is needed.